authentication-profile/README.md

```diff
@@ -0,0 +1,41 @@
+# Authentication Profile
+The Authentication Profile sample use the `DefaultAuthenticationChallengeHandler` class to take care of showing an authentication dialog for logging into a `Portal`. Once authenticated against the portal the app displays information about the authenticated user's profile.
+
+![Authentication Profile App](auth-profile.png) 
+
+## Features
+* AuthenticationManager
+* DefaultAuthenticationChallengeHandler
+* PortalInfo
+* PortalUser
+
+## Developer Pattern
+The `DefaultAuthenticationChallengeHandler` is the default implementation of the `AuthenticationChallengeHandler` interface to handle authentication with a portal.  This is set on the `AuthenticationManager`. To authenticate the user you create a `Portal` object, provide the essential credentials and then asynchronously load the portal as shown below:
+
+```java
+// Set the DefaultAuthenticationChallegeHandler to allow authentication with the portal.
+DefaultAuthenticationChallengeHandler handler = new DefaultAuthenticationChallengeHandler(this);
+AuthenticationManager.setAuthenticationChallengeHandler(handler);
+final Portal portal = new Portal("http://www.arcgis.com", true);
```

Comment:
I think we also explain the flag `loginRequired` a bit here or in the source code. When `loginRequired` set to true, the default handler will always prompt for credential while it is false the default handler will only prompt for credential when loading portals that do not allow anonymous access.